frontier/core/src/main/java/com/zhaw/frontier/screens/PauseScreen.java

```diff
@@ -0,0 +1,129 @@
+package com.zhaw.frontier.screens;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputMultiplexer;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.viewport.ExtendViewport;
+import com.zhaw.frontier.FrontierGame;
+
+/**
+ * Shows a pause screen with options to resume, save, or save and exit the game.
+ */
+public class PauseScreen implements Screen {
```

Comment:
```suggestion
public class PauseScreen extends ScreenAdapter {
```
so you can remove the empty methods